src/gsy_e/models/strategy/future/strategy.py

```diff
@@ -263,10 +265,10 @@ def event_tick(self, strategy: "BaseStrategy") -> None:
 
 def future_market_strategy_factory(
         asset_type: AssetType,
-        initial_buying_rate: float = FutureTemplateStrategiesConstants.INITIAL_BUYING_RATE,
-        final_buying_rate: float = FutureTemplateStrategiesConstants.FINAL_BUYING_RATE,
-        initial_selling_rate: float = FutureTemplateStrategiesConstants.INITIAL_SELLING_RATE,
-        final_selling_rate: float = FutureTemplateStrategiesConstants.FINAL_SELLING_RATE
+        initial_buying_rate: float = None,
+        final_buying_rate: float = None,
+        initial_selling_rate: float = None,
+        final_selling_rate: float = None
```

Comment:
I couldn't see where a user will be able to pass these variables, the calling code [here](https://github.com/gridsingularity/gsy-e/blob/7a286edadd2b336bd4b1381eaa430a916b7725c2/src/gsy_e/models/strategy/load_hours.py#L249), [here](https://github.com/gridsingularity/gsy-e/blob/7a286edadd2b336bd4b1381eaa430a916b7725c2/src/gsy_e/models/strategy/pv.py#L153) and [here](https://github.com/gridsingularity/gsy-e/blob/7a286edadd2b336bd4b1381eaa430a916b7725c2/src/gsy_e/models/strategy/storage.py#L137) are not passing them

Reply:
You are right, I noticed that as well, but thought that we are gonna need it... I will remove it! Thanks

Reply:
But for now the future strategy behaviour can only be controlled globally for all future strategies by these `constants.FutureTemplateStrategiesConstants`, which could be defined in the setup file. There is no option to set these values other way, or to define them for individual asset. So we can get rid of these attributes for now?

Reply:
I see, yes sounds good to me. Sorry for the confusion and thanks again!